components/zoho_campaigns/actions/create-campaign/create-campaign.mjs

```diff
@@ -0,0 +1,73 @@
+import app from "../../zoho_campaigns.app.mjs";
+
+export default {
+  type: "action",
+  key: "zoho_campaigns-create-campaign",
+  name: "Create Campaign",
+  version: "0.0.1",
+  description: "You can create a campaign using this API. Using this API, you can set the campaign name, subject line, sender address; choose the intended mailing list.. [See the documentation](https://www.zoho.com/campaigns/help/developers/create-campaign.html)",
+  props: {
+    app,
+    campaignname: {
+      type: "string",
+      label: "Campaign Name",
+      description: "The name of the campaign",
+    },
+    from_email: {
```

Comment:
use cammelCase

Reply:
I am using snake case to be able to use destructor in `run`. I would need to write more code if I use cammelCase for everything